lib/__tests__/extends.test.mjs

```diff
@@ -89,7 +95,7 @@ describe('extending a config from options.cwd', () => {
 			config: {
 				extends: ['./fixtures/config-string-quotes-single'],
 			},
-			cwd: __dirname,
+			cwd: fileURLToPath(new URL('.', import.meta.url)),
```

Comment:
minor

You could put it in a variable since you have it twice.
i.e. here and line 77
e.g. `const foo = new URL('.', import.meta.url);`

Reply:
I'd like to avoid `__dirname` as possible because it looks like CommonJS. I think twice occurrences are acceptable.